packages/statemanager/src/ethersStateManager.ts

```diff
@@ -405,3 +417,24 @@ export class EthersStateManager implements EVMStateManagerInterface {
     return Promise.resolve()
   }
 }
+
+export class RPCBlockChain {
+  provider: string
+  constructor(opts: {}, provider?: string) {
```

Comment:
`provider?` is marked here to be optional, but we throw if it is undefined, so this is a mandatory argument. I would also remove `opts`, it is not used? (Why does the linter not throw?)

```suggestion
  constructor(provider: string) {
```